### PR TITLE
ADD: Optional Public IP in Webhook notifications

### PR DESCRIPTION
**Pull request by apeabody**
# Description
Adds a new environmental option WEBHOOK_INCLUDE_PUBLIC_IP which adds the world visible public IP of the server to the webhook notification (default is disabled, current behavior).  Useful for servers without static IPs.

## Contributions

- I added the new feature Optional Public IP in Webhook notifications.

## Checklist

- [ ] I added one or multiple labels which best describes this PR.
- [X] I have tested the changes locally.
- [ ] This PR has a reviewer on it.
- [ ] I have validated my changes in a docker container and on **Ubuntu**. (Only needed for Odin or Docker Changes)

### CODE REVIEW

File: src/odin/notifications/mod.rs [hunk 1]
```diff
@@ -51,6 +52,39 @@ fn is_webhook_enabled() -> bool {
...
+      get_server_name(),
+      " (".to_string(),
+      get_public_ip(),
+      ")".to_string(),
+    ]
+    .join("");
+  }
+  get_server_name()
+}
+
```
Comment (mbround18) [inline src/odin/notifications/mod.rs:87]: Lets try and rip tokio out of there please, thats a large lib to include just for one http request. Use the already included lib of reqwest via something like this:

```rust

let body = reqwest::blocking::get("https://api.ipify.org?format=json").unwrap()
    .text().unwrap();
```

File: src/odin/Cargo.toml [hunk 2]
```diff
@@ -47,6 +47,8 @@ fs_extra = "1.2.0"
 glob = "0.3.0"
 a2s = "0.5.0"
 users = "0.11.0"
+tokio = { version = "1", features = ["full"] }
```
Comment (mbround18) [inline src/odin/Cargo.toml:50]: Please see https://docs.rs/reqwest/0.11.9/reqwest/blocking/index.html

**Review by mbround18 — CHANGES_REQUESTED** [review]
Hey thanks for the PR! I have included some comments.

**Comment by apeabody** [comment]
Thanks @mbround18! I was able to greatly simplify as suggested using `reqwest::blocking::get("https://api.ipify.org")`.

**Comment by mbround18** [comment]
Hey @apeabody, for some reason github isnt applying the repo secrets to your branch. Ill merge this into a repo specific branch and give it a while.

**Pull request merged**